earthpy/plot.py

```diff
@@ -0,0 +1,471 @@
+"""
+The ``earthpy`` spatial module provides functions that wrap around ``rasterio``
+and ``geopandas`` to work with raster and vector data in Python.
+"""
+
+import numpy as np
+import numpy.ma as ma
+import matplotlib.pyplot as plt
+from matplotlib import patches as mpatches
+from matplotlib.colors import ListedColormap
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import rasterio as rio
+from skimage import exposure
+import earthpy.spatial as es
+
+
+def colorbar(mapobj, size="3%", pad=0.09, aspect=20):
+    """Adjusts the height of a colorbar to match the axis height. Note that
+    this function will not work properly using matplotlib v 3.0.0 in Jupyter
+    or when exporting an image. Be sure to update to 3.0.1.
+
+    Parameters
+    ----------
+    mapobj : the matplotlib axes element.
+    size : char
+        The percent width of the colorbar relative to the plot. default = 3%
+    pad : int
+        The space between the plot and the color bar. Default = .09
+
+    Returns
+    -------
+    matplotlib.pyplot.colorbar
+
+        Matplotlib color bar object with the correct width that matches the
+        y-axis height.
+
+    Examples
+    --------
+
+    .. plot::
+
+        >>> import matplotlib.pyplot as plt
+        >>> import rasterio as rio
+        >>> import earthpy.plot as ep
+        >>> from earthpy.io import path_to_example
+        >>> with rio.open(path_to_example('rmnp-dem.tif')) as src:
+        ...     dem = src.read()
+        ...     fig, ax = plt.subplots(figsize = (10, 5))
+        >>> im = ax.imshow(dem.squeeze())
+        >>> ep.colorbar(im)  #doctest: +ELLIPSIS
```

Comment:
way to tell doctest to check that a "colorbar (in this case) is returned at memor address ....